galaxy/main/models/collection.py

```diff
@@ -89,9 +89,9 @@ class CollectionVersion(mixins.TimestampsMixin, pulp_models.Content):
     # References
     collection = models.ForeignKey(
         Collection, on_delete=models.PROTECT, related_name='versions')
-    _content = models.OneToOneField(
-        pulp_models.Content, on_delete=models.CASCADE, parent_link=True,
-        related_name='+', db_column='id',
+    _parent_ref = models.OneToOneField(
```

Comment:
this says `_parent_ref` but the migration says `parent_ref`, no underscore.

Reply:
It is `_parent_ref` with underscore in both cases. Am I missing something?

Reply:
huh... obviously i was _really_ sure i did not see an underscore there yesterday, but clearly i cannot read.